src/Selenium2Library/keywords/_tableelement.py

```diff
@@ -73,6 +73,37 @@ def table_cell_should_contain(self, table_locator, row, column, expected, loglev
             self.log_source(loglevel)
             raise AssertionError(message)
 
+    def table_cell_should_not_contain(self, table_locator, row, column, expected, loglevel='INFO'):
+        """
+        Verifies that a certain cell in a table does not contain `expected`.
+
+        Row and column number start from 1. This keyword passes if the
+        specified cell does not contain the given content. If you want
+        to test that the cell content does not match exactly, or that
+        it e.g. starts with some text, use `Get Table Cell` keyword in
+        combination with built-in keywords such as `Should Not Be Equal`
+        or `Should Not Start With`.
+
+        To understand how tables are identified, please take a look at
+        the `introduction`.
+
+        See `Page Should Contain` for explanation about `loglevel` argument.
+        """
+        message = ("Cell in table '%s' in row #%s and column #%s "
+                   "should not have contained text '%s'."
+                   % (table_locator, row, column, expected))
+        try:
+            content = self.get_table_cell(table_locator, row, column, loglevel='NONE')
+        except AssertionError, err:
+            self._info(err)
+            self.log_source(loglevel)
+            raise AssertionError(message)
```

Comment:
The exception text is not correct. It should say that: 
`Cell in table '%s' in row #%s and column #%s was not found `

But the text is correct for the exception in line 105.